swarm/reference/create.md

```diff
@@ -15,6 +15,8 @@ Later, when you use [`manage`](manage.md) or [`join`](join.md) to create Swarm m
 
 Some documentation also refers to the discovery token as a *cluster_id*.
 
-> Warning: Docker Hub's hosted discovery backend is not recommended for production use. It’s intended only for testing/development.
+> **Warning**:
```

Comment:
Sorry for not having noticed this before. #2795 is now merged, and it will automatically adds"warning", "info" or "important" before the text. If you look at the result of [your change here](https://deploy-preview-3083--docker-docs.netlify.com/swarm/reference/create/), you'll notice that the note now says "Important: warning: xxxx ".

So to start fixing #2781, we need to do a bit more than just adding a line break. The idea is to make notes a bit more meaningful by adding a small title. Check this example from the bootstrap documentation where they're doing something like this.

![screen shot 2017-04-28 at 14 59 44](https://cloud.githubusercontent.com/assets/1525937/25548853/6b2d6c90-2c23-11e7-84c9-aa3a5a6de903.png)

For this particular article you need to do something like

```
> Not for production
> Docker Hub's hosted discovery backend is not recommended for production use. It’s intended only for testing/development.
```
{:.warning}

That will give you

![screen shot 2017-04-28 at 15 02 48](https://cloud.githubusercontent.com/assets/1525937/25548948/d3613422-2c23-11e7-9bec-b4aefd83821c.png)